src/Ads/AssetSuggestionsService.php

```diff
@@ -187,17 +194,48 @@ protected function get_asset_group_asset_suggestions( int $id, string $type ): a
 	/**
 	 * Get assets from specific post or term.
 	 *
-	 * @param int    $id Post or Term ID.
+	 * @param int    $id Post or Term ID, or self::HOMEPAGE_KEY_ID.
 	 * @param string $type Only possible values are post or term.
 	 *
-	 * @return array All assets available for specific term or post.
+	 * @return array All assets available for specific term, post or homepage.
 	 */
 	protected function get_wp_assets( int $id, string $type ): array {
 		if ( $type === 'post' ) {
```

Comment:
Missing either adding `* @throws Exception` or handling the Exception in the function